frontend/awx/administration/instances/InstanceDetails.tsx

```diff
@@ -67,134 +66,119 @@ export function InstanceDetailsTab(props: {
   instance: Instance;
   instanceGroups: AwxItemsResponse<InstanceGroup> | undefined;
   instanceForks: number;
-  handleToggleInstance: (instance: Instance, isEnabled: boolean) => Promise<void>;
   handleInstanceForksSlider: (instance: Instance, value: number) => Promise<void>;
   numberOfColumns?: 'multiple' | 'single' | undefined;
 }) {
   const { t } = useTranslation();
   const pageNavigate = usePageNavigate();
   const getPageUrl = useGetPageUrl();
   const activeUser = useAwxActiveUser();
-  const {
-    instance,
-    instanceGroups,
-    handleToggleInstance,
-    instanceForks,
-    handleInstanceForksSlider,
-  } = props;
+  const { instance, instanceGroups, instanceForks, handleInstanceForksSlider } = props;
   const toolTipMap: { [item: string]: string } = useNodeTypeTooltip();
   const capacityAvailable = instance.cpu_capacity !== 0 && instance.mem_capacity !== 0;
 
   return (
-    <>
-      <PageDetails numberOfColumns={props.numberOfColumns} disableScroll>
-        <PageDetail label={t('Name')} data-cy="name">
+    <PageDetails numberOfColumns={props.numberOfColumns} disableScroll>
+      <PageDetail label={t('Name')} data-cy="name">
+        <Button
+          variant="link"
+          isInline
+          onClick={() =>
+            pageNavigate(AwxRoute.InstanceDetails, {
+              params: { id: instance.id },
+            })
+          }
+        >
+          {instance.hostname}
+        </Button>
+      </PageDetail>
+      <PageDetail label={t('Node type')} data-cy="node-type">
+        <Tooltip content={toolTipMap[instance.node_type]}>
+          <Dotted>{`${capitalizeFirstLetter(instance.node_type)}`}</Dotted>
+        </Tooltip>
+      </PageDetail>
+      <PageDetail label={t('Status')} data-cy="node-status">
+        <StatusCell status={instance.health_check_pending ? 'running' : instance.node_state} />
+      </PageDetail>
+      {instanceGroups && instanceGroups.results.length > 0 && (
+        <PageDetail label={t(`Instance groups`)} data-cy="instance-groups">
+          {instanceGroups.results.map((instance) => (
+            <Label color="blue" style={{ marginRight: '10px' }} key={instance.id}>
+              <Link to={getPageUrl(AwxRoute.InstanceGroupDetails, { params: { id: instance.id } })}>
+                {/* eslint-disable-next-line i18next/no-literal-string */}
+                {instance.name}
+              </Link>
+            </Label>
+          ))}
+        </PageDetail>
+      )}
+      {instance.related?.install_bundle && (
```

Comment:
Could you please add the tighten validation for Download bundle? like [this](https://github.com/ansible/awx/commit/40150a2be840aa75537165237ee398a38e01e96a#diff-7e68ea2512bd483497e12ce9e359fec5c3c1819aceabc3155be368330dba4217R212)

Reply:
done